engine/api/api_routes.go

```diff
@@ -405,6 +416,7 @@ func (api *API) InitRouter() {
 
 	// SSE
 	r.Handle("/events", ScopeNone(), r.GET(api.eventsBroker.ServeHTTP))
+	r.Handle("/ws", ScopeNone(), r.GET(api.websocketBroker.ServeHTTP))
```

Comment:
It is probably not so easy but we absolutely need unit test, i remember we had so much trouble with eventsBroker panics every where.

We have to check, connection, disconnection from the web socket and checks that all the references are cleaned on the broker side.
Also all the suscription and the filter feature.